R/tinyplot.R

```diff
@@ -536,6 +536,10 @@ tinyplot.default = function(
     ...
     ) {
 
+  # save for plt_add()
+  options(tinyplot_last_call = match.call(tinyplot, 
```

Comment:
To be consistent with the rest of the package infrastructure, let's rather save this to the existing `.tinyplot_env`. This is what we use in `R/get_saved_par.R`, for an example:

https://github.com/grantmcdermott/tinyplot/blob/02d78063aa25aef004a49357b9632c71957037b3/R/get_saved_par.R#L104-L116

We'll need to assign NULL values to this call object at startup in `R/zzz.R` too, e.g.
https://github.com/grantmcdermott/tinyplot/blob/02d78063aa25aef004a49357b9632c71957037b3/R/zzz.R#L46

Reply:
Unfortunately, I was unable to get this to work. I kept getting an error in the `tinyplot.density` tests. The message is obscure, having to do with a `do.call` and a `substitute`. I just can figure this out.

My guess is that this will disappear by itself when we refactor the density code.

In the meantime, I left TODOs in both files with the appropriate save code that we can eventually.

Feel free to take a shot at it if you want, but I have to give up for now.